tests/test_ostinato.py

```diff
@@ -137,3 +137,83 @@ def test_deterministic_ostinatoed_with_isconstant(seed, dask_cluster):
         npt.assert_almost_equal(ref_radius, comp_radius)
         npt.assert_almost_equal(ref_Ts_idx, comp_Ts_idx)
         npt.assert_almost_equal(ref_subseq_idx, comp_subseq_idx)
+
+
+@pytest.mark.parametrize(
+    "seed", np.random.choice(np.arange(10000), size=25, replace=False)
+)
+def test_input_not_overwritten(seed):
+    m = 50
+    np.random.seed(seed)
+    Ts = [np.random.rand(n) for n in [64, 128, 256]]
+
+    # without nan values
```

Comment:
@seanlaw 
Do we need to check this scenario, i.e. having no `nan` value in the original input ?

Reply:
I had no intentions to. Is it necessary?

Reply:
Not really. Initially I thought `ostinato` may change some values of the input time series to `nan` as it searches through data to find the consensus. I took a look and noticed that does not happen in the code (and in fact, you already mentioned that in #982).  I am going to remove it.